moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -153,6 +164,22 @@ const DashboardView = () => {
     />
   );
 
+  /** Build custom user greeting
+   */
+  const date = moment();
+  const curHr = date.format("HH");
+  const dateFormatted = date.format("dddd - MMMM DD, YYYY");
+
+  const getTimeOfDay = curHr => {
+    if (curHr < 12) {
+      return "morning";
+    } else if (curHr < 18) {
+      return "afternoon";
+    } else {
+      return "evening";
+    }
```

Comment:
this works fine, but this is a good use case for the js `switch` statement. Consider using it when you have 3+ conditions. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch